recipe/meta.yaml

```diff
@@ -19,17 +19,23 @@ requirements:
     - python {{ python_min }}
     - pip
     - pytest-runner
+    - setuptools >=61
+    - setuptools-scm >=8
   run:
     - python >={{ python_min }}
     - attrs
     - pandas
+    - pint
```

Comment:
Needed according to:
https://github.com/ecmwf/pdbufr/blob/0.14.0/pyproject.toml#L33